internal/runners/branch/tree.go

```diff
@@ -0,0 +1,192 @@
+package branch
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"sort"
+
+	"github.com/ActiveState/cli/pkg/platform/api/mono/mono_models"
+	"github.com/go-openapi/strfmt"
+)
+
+type branchNode struct {
+	Label string
+	ID    string
+}
+
+type tree map[branchNode]tree
+
+type BranchTree struct {
+	tree                  tree
+	branches              mono_models.Branches
+	rootBranches          mono_models.Branches
+	localBranch           string
+	branchFormatting      string
+	localBranchFormatting string
+}
+
+const (
+	edgeLink string = "│  "
+	edgeMid  string = "├─"
+	edgeEnd  string = "└─"
```

Comment:
It's kinda strange that you have functions to set formatting but then this part is hardcoded. Either hardcore or don't (I'd say for our use case we should just hard code as this will be thrown away when we revisit formatting).